README.md

```diff
@@ -1,7 +1,119 @@
-# givit
-## team members:
+<h1 align="center">
+  <img src="../givit/static/img/leaveit2givit.jpg" />
```

Comment:
Looks like this didn't work - try to check in the [Guide](https://guides.github.com/features/mastering-markdown/) how to add an image to markdown.